packages/core/src/Provider/NetworkInfoProvider/types.ts

```diff
@@ -13,6 +13,7 @@ export type StakeSummary = {
 
 export type NetworkInfo = {
   network: {
+    id: Cardano.CardanoNetworkId;
```

Comment:
There is already a `Cardano.NetworkId` enum. Let's just use that.

Reply:
This was done to avoid additional mapping between `Cardano.NetworkId` ( only small letters enum ) and the `[networkId](https://github.com/input-output-hk/cardano-configurations/blob/568756920ef0249b9cee09e0b973c2aacec241de/network/testnet/genesis/shelley.json#L58)` values extracted from cardano-configurations ( starts with a capital letter )

Reply:
I would prefer an additional mapping over 2 core types representing the same thing. The value in the original type is a number (`0` or `1`).

Reply:
addressed [e184729](https://github.com/input-output-hk/cardano-js-sdk/pull/255/commits/e18472906a14c6ab80ca1fbe6eaaf260fd85b1b6)